panel/src/components/Views/Users/UserAvatar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<k-dropdown>
+	<div>
```

Comment:
Is the div here really needed?

Reply:
Since Vue SFC can only have a single element as root, yes.

Reply:
🙈 of course.